packages/lib/src/index.ts

```diff
@@ -7,6 +7,7 @@ import getTheme from './getTheme';
 import { allEqual } from './allEqual';
 export { validateChildren } from './validateChildren';
 export { createSyntheticEvent } from './createSyntheticEvent';
+import isObject from 'lodash/isObject';
```

Comment:
This should have been caught by the linter, but `imports` should come before `exports`